README.md

```diff
@@ -1,9 +1,14 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# React-Capstone 
+# Metrics webapp 
 
```

Comment:
- [ ] The are some lapses in the alternating dual-color pattern of your design. Kindly note that we as much as possible want your design to match the provided [template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Therefore kindly ensure that cells of your list follow the **alternating dual-color pattern**. Each cell of the list item should have surrounding cells having **different (alternating) background color(s)**.


**Lapses in your design:**
![image](https://user-images.githubusercontent.com/12745474/147232074-784874c6-23d1-43ee-9998-0d15edba3d8b.png)


**The original design:**
![image](https://user-images.githubusercontent.com/12745474/142270463-4d16b9d3-7557-430c-80eb-a7c40589a4a6.png)